pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -460,6 +461,30 @@ def get_jobid(self, handle: ResultHandle) -> str:
         """
         return cast(str, handle[0])
 
+    @staticmethod
+    def get_ppcirc_rep(handle: ResultHandle) -> Any:
+        """Return the JSON serialization of the classiocal postprocessing circuit
+        attached to a handle, if any.
+
+        :param handle: result handle
+        :return: serialized post-processing circuit, if any
+        """
+        return json.loads(cast(str, handle[1]))
+
+    @staticmethod
+    def get_results_width(handle: ResultHandle) -> Optional[int]:
+        """Return the truncation width of the results, if any.
+
+        :param handle: result handle
```

Comment:
documentation of the type is missing

Reply:
I prefer not to add it to the docstring as it's already in the function signature (which is visible in the published docs), so it would just be duplicating information.